### DISCUSSION TITLE
Comment ajouter un bouton d'actualisation (refresh) ?

### DISCUSSION THREAD

**Discussion by elyancolin** [witsa/synapps#623] [Q&A]

**Comment by WITNicolas** [comment]
Bonjour,

Il n'y a (en théorie) pas besoin de bouton refresh, le rafraichissement se fait lorsque les acteurs liés à une source de données générent une requête ou bien au moment où on affiche une scene.
Il faut éventuellement régler les tempos de refresh soit au niveau des acteurs Source de Ressources, des Acteurs Détails de Reflets ou bien au niveau du Global/Ressource dans le menu PROJET / FOURNISSEUR DE DONNEE.
(c'est selon la méthode que vous avez employé)

Plus précisement quel serait le besoin autour du bouton Refresh ?

**Comment by elyancolin** [comment]
Bonjour,

je dois gérer des pompes et des aérothermes dans mon synnap's malheureusement quand leurs états change par exemple que je force une mache sur une pompe je suis obligé de changer de page et revenir sur cette dernière pour pouvoir visualiser  la pompe en marche.
J'aurai donc aimer pouvoir avoir un bouton qui actualiserai la page pour que je n'ai plus à effectuer ce changement de pages.
Cela est-il possible ?
